src/MusicalScore/VoiceData/SourceMeasure.ts

```diff
@@ -214,7 +215,7 @@ export class SourceMeasure extends BaseIdClass {
                 }
             }
         }
-        //log.debug("created new container: ", staffEntry, this.verticalSourceStaffEntryContainers);
+        log.trace("created new container: ", staffEntry, this.verticalSourceStaffEntryContainers);
```

Comment:
Should be debug (trace always includes the whole stack which would result in shitloads of output information). BUT I think this should be removed also as we are done implementing voice entries and ghostnotes

Reply:
yes, better remove it - we just needed it for debugging